### ISSUE TITLE
Clause template buttons overlap tooltips

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/web-components#185]
# Bug Report 🐛
Clause template buttons overlap tooltips so that the tooltips can't be read, like this example:
![image](https://user-images.githubusercontent.com/40569793/92005485-892ee400-ed11-11ea-8138-c084f5cf1a43.png)

## Expected Behavior
Tooltips should appear in front of everything else

**Comment by Manan17** [comment]
It just needs a z-index = 1 in css file so that the tooltips can be read easily

**Comment by irmerk** [comment]
@Manan17 yeah I think that makes sense. I believe a lot of other components in there use `z-index`, so it may need to be larger than `1`.

**Comment by Manan17** [comment]
Can I work on this issue?

**Comment by Michael-Grover** [comment]
@Manan17 yes, thanks! I've assigned the issue to you.

**Comment by Manan17** [comment]
I cannot see the clause template buttons in the live demo. Is it in the original code? So I have to run the app locally?

**Comment by Michael-Grover** [comment]
@Manan17 I recorded how to replicate the issue, let me know if this helps.

https://www.loom.com/share/fb5889cfc04a405f9a68c9212ea3f728

You have to go to the storybook https://ap-web-components.netlify.app/?path=/story/*
then select the contract editor
then choose the Late Delivery and Penalty template in the Knobs section

**Comment by Michael-Grover** [comment]
in the video I chose the wrong template, but if you choose "Late Delivery and Penalty (with optional & this)" you will see the issue

![image](https://user-images.githubusercontent.com/40569793/93250071-3d902780-f760-11ea-80f2-882fada5d3bd.png)

**Comment by Manan17** [comment]
Thanks. It helped me , starting to work on the issue

**Comment by K-Kumar-01** [comment]
@Michael-Grover @irmerk 
I have opened a PR for the issue as the issue seemed inactive to me.
 Let me know if there are any changes required.
